### PR TITLE
Support build wasm

### PR DESCRIPTION
**Pull request by nploi**
fixed: https://github.com/kevmoo/peanut.dart/issues/176

### CODE REVIEW

File: lib/src/flutter.dart [hunk 1]
```diff
@@ -22,8 +22,12 @@ Future<void> runFlutterBuild(
   final args = <String>[
     'build',
     'web',
-    '--web-renderer',
-    options.webRendererString(),
+    if (options.wasm) ...[
+      '--wasm',
+    ] else ...[
```
Comment (kevmoo) [inline lib/src/flutter.dart:27]:
```suggestion
    if (options.wasm) '--wasm',
    else ...[
```

Does this work?

Comment (nploi) [reply lib/src/flutter.dart]: yes.

**Comment by nploi** [comment]
hi @kevmoo, can you help to review again ?

File: lib/src/flutter.dart (hunk 1 above)
Comment (kevmoo) [reply lib/src/flutter.dart]: thanks!

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+## 5.0.2
+
+- Support build wasm option for Flutter web.
+
+
```
Comment (kevmoo) [inline CHANGELOG.md:5]: delete extra line

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,8 @@
+## 5.0.2
```
Comment (kevmoo) [inline CHANGELOG.md:1]:
```suggestion
## 5.1.0
```

It's a feature!

File: pubspec.yaml [hunk 4]
```diff
@@ -1,5 +1,5 @@
 name: peanut
-version: 5.0.1-wip
+version: 5.0.2
```
Comment (kevmoo) [inline pubspec.yaml:2]:
```suggestion
version: 5.1.0
```

adding a feature!

**Review by kevmoo — CHANGES_REQUESTED** [review]

**Comment by nploi** [comment]
fixed your comments @kevmoo

**Review by kevmoo — APPROVED** [review]

**Comment by kevmoo** [comment]
@nploi – you'll need to run the build again to update the version file and we'll be good!

**Comment by nploi** [comment]
> @nploi – you'll need to run the build again to update the version file and we'll be good!

@kevmoo  done  🙏

**Pull request merged**